source/developers-guide/code-snippets/index.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: default
+title: Code snippet collection
+github_link: developers-guide/code-snippets/index.md
+shopware_version: 5.2.0
+indexed: true
+---
+
+This article will provide some useful code snippets that can help you when developing Shopware plugins. Please feel free to add additional snippets via GitHub!
+
+## Read  config from a different shop than the current one
+
+```php
+/** @var \Shopware_Components_Config $config */
+$config = \Shopware()->Container()->get('config');
+$config->setShop($shop);
```

Comment:
Absolutely! I've just chosen this example because I have asked in the IRC how to get shop specific context which was the origin of the idea to collect code snippets in the devdocs. As soon as I've got an idea for a better example, this code will be code.